[2025-07-29T22:16:27.542+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: table_dag.submit_tablea scheduled__2025-03-26T00:00:00+00:00 [queued]>
[2025-07-29T22:16:27.551+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: table_dag.submit_tablea scheduled__2025-03-26T00:00:00+00:00 [queued]>
[2025-07-29T22:16:27.552+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2025-07-29T22:16:27.563+0000] {taskinstance.py:1380} INFO - Executing <Task(PinotTableSubmitOperator): submit_tablea> on 2025-03-26 00:00:00+00:00
[2025-07-29T22:16:27.569+0000] {standard_task_runner.py:57} INFO - Started process 1245 to run task
[2025-07-29T22:16:27.571+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'table_dag', 'submit_tablea', 'scheduled__2025-03-26T00:00:00+00:00', '--job-id', '747', '--raw', '--subdir', 'DAGS_FOLDER/table_dag.py', '--cfg-path', '/tmp/tmpgjfhs3ly']
[2025-07-29T22:16:27.572+0000] {standard_task_runner.py:85} INFO - Job 747: Subtask submit_tablea
[2025-07-29T22:16:27.610+0000] {task_command.py:415} INFO - Running <TaskInstance: table_dag.submit_tablea scheduled__2025-03-26T00:00:00+00:00 [running]> on host 931bde20eb1e
[2025-07-29T22:16:27.674+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='GGCODE' AIRFLOW_CTX_DAG_ID='table_dag' AIRFLOW_CTX_TASK_ID='submit_tablea' AIRFLOW_CTX_EXECUTION_DATE='2025-03-26T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-26T00:00:00+00:00'
[2025-07-29T22:16:27.687+0000] {pinot_table_operator.py:28} ERROR - Pinot table failed to submit to Apache Pinot! 409 - {"code":409,"error":"Table config for account_dim_OFFLINE already exists. If this is unexpected, try deleting the table to remove all metadata associated with it."}
[2025-07-29T22:16:27.688+0000] {pinot_table_operator.py:32} ERROR - An error occured: Table submission failed with status code 409
[2025-07-29T22:16:27.695+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=table_dag, task_id=submit_tablea, execution_date=20250326T000000, start_date=20250729T221627, end_date=20250729T221627
[2025-07-29T22:16:27.709+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-07-29T22:16:27.727+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
